app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Editing an Article</h1>
```

Comment:
- [ ] When I tried to `reset` my password it's breaking the app. Based on the requirement, user is able to reset their password. Also enable in user model `:confirmable` for device.